packages/terra-application-header-layout/tests/wdio/application-header-spec.js

```diff
@@ -15,21 +12,19 @@ Terra.describeViewports('Application Header Layout', ['tiny', 'medium'], () => {
       browser.keys(['Tab']);
     });
 
-    Terra.it.validatesElement('Visible', { selector: '#test-header' });
+    it('should be visible', () => {
+      Terra.validates.element('Visible', { selector: '#test-header' });
+    });
 
     it('should hide the button after a tab', () => {
       browser.keys(['Tab', 'Tab']);
+      Terra.validates.element('Not Visible', { selector: '#test-header' });
```

Comment:
Merge these and remove the before hook. Before hooks are inefficient because you will load the url for each it block when you only need to load it once.

Reply:
Done d17aaa4